gensim/models/wrappers/wordrank.py

```diff
@@ -95,14 +95,17 @@ def train(cls, wr_path, corpus_file, out_path, size=100, window=15, symmetric=1,
         cmd_del_vocab_freq = ['cut', '-d', " ", '-f', '1', temp_vocab_file]
 
         commands = [cmd_vocab_count, cmd_cooccurence_count, cmd_shuffle_cooccurences]
-        logger.info("Prepare training data using glove code '%s'", commands)
         input_fnames = [corpus_file.split('/')[-1], corpus_file.split('/')[-1], cooccurrence_file]
         output_fnames = [temp_vocab_file, cooccurrence_file, cooccurrence_shuf_file]
 
+        logger.info("Prepare training data using glove code")
         for command, input_fname, output_fname in zip(commands, input_fnames, output_fnames):
+            logger.info("Prepare '%s' using '%s'", output_fname, command)
             with smart_open(input_fname, 'rb') as r:
                 with smart_open(output_fname, 'wb') as w:
                     utils.check_output(w, args=command, stdin=r)
+
+        logger.info("Delete frequencies from vocab file")
```

Comment:
It may be necessary to add logging arguments to the function `check_output`, @tmylk what do you think about it?